src/Microsoft.AspNet.Authentication/DataHandler/TicketDataFormat.cs

```diff
@@ -7,7 +7,8 @@ namespace Microsoft.AspNet.Authentication
 {
     public class TicketDataFormat : SecureDataFormat<AuthenticationTicket>
     {
-        public TicketDataFormat(IDataProtector protector) : base(new TicketSerializer(), protector)
+        public TicketDataFormat(IDataProtector protector)
+            : base(TicketSerializer.Default, protector)
```

Comment:
Not for `PropertiesSerializer`:

https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.Twitter/Messages/RequestTokenSerializer.cs#L98

https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication/DataHandler/TicketSerializer.cs#L66